lib/app.coffee

```diff
@@ -33,6 +33,9 @@ events = require('./events')
 plugins = require('./utils/plugins')
 update = require('./utils/update')
 
+# Assign bluebird as the global promise library
+require('any-promise/register/bluebird')
```

Comment:
Ah, ok. There is https://www.npmjs.com/package/stream-to-promise-agnostic, but it looks like a fork that's falling behind their master, and I don't think it's really worth the effort. Fine to ignore this :+1: